### PR TITLE
add more context to ent permissions table

### LINKED ISSUE #5742: Clarify levels in RBAC permissions table
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

@b-per shared some feedback about how the RBAC docs aren't easy to understand. He shared the attached screenshot asking for more information about what those permissions really mean for dbt Cloud users.

image

For example, what does the `R/W` access to “dbt adapters” mean? And does “Credentials” refer to Env credentials or Dev credentials or both?

Raised this in [internal slack](https://dbt-labs.slack.com/archives/C02N953LRDF/p1720176389355509) for confirmation

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/docs/cloud/manage-access/enterprise-permissions

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr seeks to clarify to the enterprise cloud ui permissions table. raised by @b-per in the internal slack: https://dbt-labs.slack.com/archives/C02NCQ9483C/p1720167650879159

originally raised by @b-per 

Resolve #5742 however, there are future improvements needed based on https://github.com/dbt-labs/docs.getdbt.com/issues/5752

### CODE REVIEW

**Comment by mirnawong1** [comment]
related to #5742

File: website/snippets/_enterprise-permissions-table.md [hunk 1]
```diff
@@ -35,9 +35,9 @@ Account roles enable you to manage the dbt Cloud account and manage the account
 |Project-level permission | Account Admin | Billing admin | Project creator | Security admin | Viewer |
```
Comment (mirnawong1) [inline website/snippets/_enterprise-permissions-table.md:35]: is the 'project-level permission' column meant to reference dbt cloud UI pages?

File: website/snippets/_enterprise-permissions-table.md [hunk 2]
```diff
@@ -75,9 +75,9 @@ The project roles enable you to work within the projects in various capacities.
 |Project-level permission  | Admin | Analyst | Database admin | Developer | Git Admin | Job admin | Job runner  | Job viewer  | Metadata <br></br> (Discovery API only) | Semantic Layer | Stakeholder | Team admin | Webhook |
 |--------------------------|:-----:|:-------:|:--------------:|:---------:|:---------:|:---------:|:-----------:|:-----------:|:--------:|:--------------:|:-----------:|:----------:|:-------:|  
 | Connections              |   W   |    R    |       W        |     R     |     R     |     R     |             |             |          |                |     R       |     R      |         |
```
Comment (mirnawong1) [inline website/snippets/_enterprise-permissions-table.md]: what specific "connections" is this? git or data platform connection,etc.?

File: website/snippets/_enterprise-permissions-table.md [hunk 3]
```diff
@@ -31,31 +31,31 @@ Account roles enable you to manage the dbt Cloud account and manage the account
...
-| Credentials             |       W       |               |       W         |                |   R    |
+| Environment credentials             |       W       |               |       W         |                |   R    |
 | Custom env. variables   |       W       |               |       W         |                |   R    |
-| dbt adapters            |       W       |               |       W         |                |   R    |
+| Data platform config            |       W       |               |       W         |                |   R    |
 | Develop (IDE or dbt Cloud CLI)           |       W       |               |       W         |                |        |
 | Environments            |       W       |               |       W         |                |   R    |
 | Jobs                    |       W       |               |       W         |                |   R    |
-| Metadata                |       R       |               |       R         |                |   R    |
+| Metadata                 |       R       |               |       R         |                |   R    |
```
Comment (mirnawong1) [inline website/snippets/_enterprise-permissions-table.md]: what does metadata mean here?

File: website/snippets/_enterprise-permissions-table.md [hunk 4]
```diff
@@ -71,20 +71,20 @@ The project roles enable you to work within the projects in various capacities.
...
+| Data platform config             |   W   |    W    |       W        |     W     |     R     |     W     |             |             |          |                |     R       |     R      |         |
 | Develop <br />(IDE or dbt Cloud CLI)            |   W   |    W    |                |     W     |           |           |             |             |          |                |             |            |         |
 | Environments             |   W   |    R    |       R        |     R     |     R     |     W     |             |      R      |          |                |     R       |     R      |         |
 | Jobs                     |   W   |    R    |       R        |     W     |     R     |     W     |      R      |      R      |          |                |     R       |     R      |         |
-| Metadata                 |   R   |    R    |       R        |     R     |     R     |     R     |             |      R      |     R    |                |     R       |     R      |         |
+| Metadata                  |   R   |    R    |       R        |     R     |     R     |     R     |             |      R      |     R    |                |     R       |     R      |         |
```
Comment (mirnawong1) [inline website/snippets/_enterprise-permissions-table.md]: what does metadata mean here?

File: website/snippets/_enterprise-permissions-table.md [hunk 5]
```diff
@@ -31,31 +31,31 @@ Account roles enable you to manage the dbt Cloud account and manage the account
 | Webhooks                |     W         |               |                 |                |        |
 
 #### Project permissions for account roles
-
+ 
 |Project-level permission | Account Admin | Billing admin | Project creator | Security admin | Viewer | 
 |:-------------------------|:-------------:|:-------------:|:---------------:|:--------------:|:------:| 
 | Connections             |       W       |               |       W         |                |   R    |
-| Credentials             |       W       |               |       W         |                |   R    |
+| Environment credentials             |       W       |               |       W         |                |   R    |
```
Comment (mirnawong1) [inline website/snippets/_enterprise-permissions-table.md]: by env credentials, do we mean the deployment credentials page?

File: website/snippets/_enterprise-permissions-table.md (hunk 1 above)
Comment (gcs272) [reply website/snippets/_enterprise-permissions-table.md:35]: I'm not totally sure I understand. Can you say more about the link to cloud ui? FWIW, I think the text on :35 could just be `permission`.

File: website/snippets/_enterprise-permissions-table.md (hunk 5 above)
Comment (gcs272) [reply website/snippets/_enterprise-permissions-table.md]: I think so, yep

File: website/snippets/_enterprise-permissions-table.md (hunk 3 above)
Comment (gcs272) [reply website/snippets/_enterprise-permissions-table.md]: I'm slightly iffy on metadata permissions - any chance someone else wants to chime in here?

File: website/snippets/_enterprise-permissions-table.md (hunk 2 above)
Comment (gcs272) [reply website/snippets/_enterprise-permissions-table.md]: iirc it's strictly warehouse config

File: website/snippets/_enterprise-permissions-table.md (hunk 1 above)
Comment (mirnawong1) [reply website/snippets/_enterprise-permissions-table.md:35]: oh sorry @gcs272 , i meant to ask if the items under the 'project level permissions' column are meant to specifically map to dbt cloud UI pages? e..g connections is the environment data platform connections page, credentials is the env credentials page, etc.)

Screenshot 2024-07-09 at 09 29 34

File: website/snippets/_enterprise-permissions-table.md (hunk 3 above)
Comment (mirnawong1) [reply website/snippets/_enterprise-permissions-table.md]: no worries! asked here https://dbt-labs.slack.com/archives/C02N953LRDF/p1720515202761889?thread_ts=1720176389.355509&cid=C02N953LRDF

Comment (mirnawong1) [reply website/snippets/_enterprise-permissions-table.md]: hey @gcs272 , Topher confirmed that essentially 'metadata' here means permissions query the metadata graphql api

File: website/snippets/_enterprise-permissions-table.md (hunk 1 above)
Comment (gcs272) [reply website/snippets/_enterprise-permissions-table.md:35]: ah, I don't think they're really supposed to be 1:1, but it's pretty close

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thanks for flagging this. The PR is great. The staging example, is really good and explains the permissions perfectly. I'll approve this for you.

Kind Regards

**Pull request merged**